model_logging/migrations/0001_initial.py

```diff
@@ -21,7 +20,7 @@ class Migration(migrations.Migration):
                 ('date_created', models.DateTimeField(default=django.utils.timezone.now)),
                 ('operation', models.CharField(max_length=255, choices=[('added', 'Added'), ('removed', 'Removed'), ('modified', 'Modified')])),
                 ('model_path', models.CharField(max_length=255)),
-                ('data', pgcrypto.fields.TextPGPPublicKeyField(default='')),
```

Comment:
That would be cool, but no pressure -- after all, for all I know, you're the only users of this lib `:)`